skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -178,7 +177,7 @@ def _nl_means_denoising_2d(image, int s=7, int d=13, float h=0.1):
     cdef int row_start, row_end, col_start, col_end
     cdef int row_start_i, row_end_i, col_start_j, col_end_j
     cdef IMGDTYPE [::1] new_values = np.zeros(n_ch).astype(np.float32)
-    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(util.pad(image,
+    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(np.pad(image,
```

Comment:
I really do think we should endeavor to expose full functionality wherever possible. Having `crop` but not `pad` is strange. Assuming users can figure out where to go for their padding needs exterior to the package is not ideal either (earlier this week I met another person who implemented their own 2-d padding because they didn't know about np.pad). 

So I don't think this should go away.